pkg/docker/util.go

```diff
@@ -286,6 +286,28 @@ func GetBuilderImage(config *api.Config) (*PullResult, error) {
 	return r, nil
 }
 
+// GetRebuildImage obtains the metadata information for the image
+// specified in a s2i rebuild operation.  Assumptions are made that
+// the build is available locally since it should have been previously built.
+func GetRebuildImage(config *api.Config) (*PullResult, error) {
+	d, err := New(config.DockerConfig, config.PullAuthentication)
+	if err != nil {
+		return nil, err
+	}
+
+	r, err := PullImage(config.Tag, d, api.PullNever, false)
```

Comment:
i was making the assumption that it was always local  ... if we can't make that assumption, do we repurpose the builder pull policy or previous image pull policy, or do we have to add another field in the config for rebuild pull policy ?